app/src/page/subscriptions/view.js

```diff
@@ -55,6 +62,24 @@ class SubscriptionsPage extends React.PureComponent {
       <View style={subscriptionsStyle.container}>
 
       {hasSubscriptions && !loading &&
+      <View style={subscriptionsStyle.viewModeRow}>
+        <Link
+          text={'All Subscriptions'}
+          style={[subscriptionsStyle.viewModeLink,
+                  ((viewMode === Constants.SUBSCRIPTIONS_VIEW_ALL) ? subscriptionsStyle.activeMode : subscriptionsStyle.inactiveMode)]}
```

Comment:
@seanyesmunt @akinwale any suggestions on a pattern we can adopt globally to simplify this?
We repeat a lot of similar logic across the board when applying styles.

Reply:
I'll have to think about this. I agree it would be nice to be able to have a way of simplifying this sort of thing. We could make use of the CSS module that @seanyesmunt recommended, but that's going to require converting all of the styles which could take some time.